pallets/dactr/src/lib.rs

```diff
@@ -122,7 +122,7 @@ pub mod pallet {
 		}
 
 		#[pallet::call_index(1)]
-		#[pallet::weight(T::WeightInfo::submit_data(data.len() as u32))]
+		#[pallet::weight(({T::WeightInfo::submit_data(data.len() as u32) + T::WeightInfo::data_root(data.len() as u32)}, DispatchClass::Normal))]
```

Comment:
Please use `saturated_add` and `saturated_into` instead of `+` or `as u32`.
What do you think about using a helper function like [this](https://github.com/availproject/avail/blob/cea467c57619e7d606a3ccdf9615c280771dcb9b/pallets/system/src/lib.rs#L1877) ?
Something like:
```Rust
#[pallet::weight(weight_helper::submit_data(data.len())]
```